### PR TITLE
eng fix skinning of osu!

### PR DESCRIPTION
**Pull request by miterosan**

### CODE REVIEW

File: wiki/Skinning/osu!/en.md [hunk 1]
```diff
@@ -1,117 +1,39 @@
...
-| ![No][false] | ![Yes][true]      | all             | Centre       | -            | Normal     |
-
-- scaled down by 0.8x
 
 ## Hitcircle
 
-### approachcircle.png
+``approachcircle.png``
 
 ![](fs/approachcircle.png)
```
Comment (sr229) [inline wiki/Skinning/osu!/en.md]: L36: Add alternate text just in case the file fails to load.

File: wiki/Skinning/osu!/en.md [hunk 2]
```diff
@@ -125,7 +47,7 @@ On this note, you can use the skin.ini file to define the path for these images
 - this element shrinks over time
 - this is the visual cue that can help a player know when to tap the hitobject
 
-### hitcircle.png
+``hitcircle.png``
 
 ![](fs/hitcircle.png)
```
Comment (sr229) [inline wiki/Skinning/osu!/en.md]: L50: Add alternate text just in case the file fails to load.

File: wiki/Skinning/osu!/en.md [hunk 3]
```diff
@@ -135,7 +57,7 @@ On this note, you can use the skin.ini file to define the path for these images
 
 1. this element is tinted with the combo colours
 
-### hitcircleoverlay.png
+``hitcircleoverlay.png``
 
 ![](fs/hitcircleoverlay.png)
```
Comment (sr229) [inline wiki/Skinning/osu!/en.md]: L62: Add alternate text just in case the file fails to load.

File: wiki/Skinning/osu!/en.md [hunk 4]
```diff
@@ -162,7 +84,7 @@ On this note, you can use the skin.ini file to define the path for these images
 - used only in the editor
 - this is the highlight for hitcircles and sliders
 
-### followpoint.png
+``followpoint.png``
 
 ![](fs/followpoint.png)
```
Comment (sr229) [inline wiki/Skinning/osu!/en.md]: L89: Add alternate text just in case the file fails to load.

File: wiki/Skinning/osu!/en.md [hunk 5]
```diff
@@ -173,7 +95,7 @@ On this note, you can use the skin.ini file to define the path for these images
 - the arrow should point towards the RIGHT
 - this stays on the screen for 1.2 seconds (1200ms)
 
-### lighting.png
+``lighting.png``
 
 ![](fs/lighting.png)
```
Comment (sr229) [inline wiki/Skinning/osu!/en.md]: L100: Add alternate text just in case the file fails to load.

**Comment by Repflez** [comment]
Closed by #436

**Pull request closed without merging**